### PR TITLE
Prep 1.3.8

### PR DESCRIPTION
**Pull request by corruptmemory**
* Restructured directory layout as per PR #1092 
* Works decently with Play 2.5
* Updated offline catalog

### CODE REVIEW

File: project/LocalTemplateRepo.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ object LocalTemplateRepo {
     libraryDependencies += Dependencies.templateCache,
     // TODO - Allow debug version for testing?
     remoteTemplateCacheUri := "http://downloads.typesafe.com/typesafe-activator",
-    localTemplateCacheHash := "b419dc7f725ca81ec67e863f5c3a7da1bbd38a58",
+    localTemplateCacheHash := "330dfd06afffb0229f43d0398cbe81fd390d3e54",
```
Comment (henrikengstrom) [inline project/LocalTemplateRepo.scala]: Gots to love this :)

File: ui-common/src/main/scala/activator/UICacheHelper.scala [hunk 2]
```diff
@@ -62,7 +62,6 @@ object UICacheHelper {
       e
     } map (_ -> name)
 
-    throw new Exception("FUCK NO!!!")
```
Comment (henrikengstrom) [inline ui-common/src/main/scala/activator/UICacheHelper.scala]: LOL

**Comment by henrikengstrom** [comment]
LGTM. Nice work @corruptmemory. Did not see any `Typesafe` -> `Lightbend` changes as per the comment above, but I guess this is fine anyhow.

**Comment by corruptmemory** [comment]
Arg (in the TS -> LB change).  I think that was actually in a previous PR.

**Pull request merged**